spec/domain/payment_provider_spec.rb

```diff
@@ -127,6 +127,45 @@
     end
   end
 
+  context 'XTC' do
+    it 'sends XTC order' do
+      response = ["01DC75AB6551FC250D84AE3298EC7C1C", "N031"]
```

Comment:
was bedeuten diese Zahlen? Ist das nur eine `ok_response`?

Reply:
Jup, um genau zu sein ist das die order_id und die transaction_id
Siehe https://github.com/railslove/epics/blob/master/lib/epics/client.rb#L226

Hab jetzt die response noch in einzelne entsprechende variabeln aufgeteilt dass es klarer ist